Readme.md

```diff
@@ -1,21 +1,38 @@
-To do list
-```
-As a personal assistant
-I can see a list of my to dos
-So I can plan my day
-```
-```
-As a personal assistant
-I can add a new to do item to my list
-So I can record the things I need to do
-```
-```
-As a personal assistant
-I can mark a to do item as complete
-So I keep track of what I've already done
-```
-```
-As a personal assistant
-I can see which to dos I have completed and which I haven't yet completed
-So I can quickly decide what to do next
+# Unicorn web framework
+
+The Unicorn framework generates a scaffold for a simple node server with a loopy template, to populate a webpage dynamically.
+
+## Installation
+1. Install [Node](https://nodejs.org/en/)
+1. Create a new directory using `mkdir <directory name>`
+1. `cd <directory name>` to change into the directory
+1. Generate a Unicorn scaffold with `npm install ../unicorn-framework`
```

Comment:
Please replace `../unicorn-framework` with (?) unicorn-lists (?)

unicorn already exists in npm so we gotta pick a different name like the one above